src/share/classes/com/sun/javatest/regtest/exec/Agent.java

```diff
@@ -90,12 +91,25 @@ public static class Fault extends Exception {
     static final boolean showAgent = Flags.get("showAgent");
     static final boolean traceAgent = Flags.get("traceAgent");
 
```

Comment:
(minor) suggest adding a comment that this is to allow running `jtreg` on older JDKs that do not support the `pid` method.

Reply:
Done - I've updated the PR to include a code comment for this line.